Philips.CodeAnalysis.Test/MsTest/DataTestMethodsHaveDataRowsAnalyzerTest.cs

```diff
@@ -102,7 +102,7 @@ public void Foo() {{ }}
 		[DataRow("", false)]
 		[DataTestMethod]
 		[TestCategory(TestDefinitions.UnitTests)]
-		public async Task TestMethodsMustNotHaveDataRowsAsync(string arg, bool isError)
+		public async Task MustNotHaveDataRowsAsync(string arg, bool isError)
```

Comment:
If there is no blocking variant, I would not postfix with `Async`. This is a result of a refactoring, which I have undone in a lot of places already.

Reply:
OK. I'll have this on the back of my mind moving forward.